packages/jest-expo/universal/jest-preset.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const reactNativePreset = require('react-native-web/jest-preset');
+const expoPreset = require('../jest-preset');
+
+const COLORS = {
+  android: 'blueBright',
+  ios: 'white',
+  web: 'magenta',
+  node: 'cyan',
+};
+
+function getModuleFileExtensions(...platforms) {
+  let fileExtensions = [];
+
+  // Support both TypeScript and JavaScript
+  for (const language of ['t', 'j']) {
```

Comment:
just write out js, jsx, ts, tsx

Reply:
This wouldn't respect the required order. [`web.ts`, `web.tsx`, `ts`, `tsx`, `web.js`, `web.jsx`, etc...]

Reply:
i mean replace `${language}s`, `${language}sx` with the extensions written out in whatever order needed (unroll the for-loop below)